### PR TITLE
Add more documentation on allowed preview flags

### PR DESCRIPTION
**Pull request by premkumr**

### CODE REVIEW

File: docs/content/preview/reference/configuration/yb-master.md [hunk 1]
```diff
@@ -983,6 +983,16 @@ expensive when the number of yb-tservers, or the number of databases goes up.
 
 ## Advanced flags
 
+##### --allowed_preview_flags_csv
+
+This is a comma-separated values (CSV) formatted catalogue of [preview feature](../../../releases/versioning/#tech-preview-tp) flag names. Preview flags represent experimental or in-development features that are not yet fully supported. Flags that are tagged as Preview cannot be modified or configured unless they are included in this list.
+
+By adding a flag to this list, you explicitly acknowledge and accept any potential risks or instability that may arise from modifying these preview features. This process serves as a safeguard, ensuring that users are fully aware of the experimental nature of the flags they are working with.
```
Comment (aishwarya24) [inline docs/content/preview/reference/configuration/yb-master.md:990]:
```suggestion
By adding a flag to this list, you explicitly acknowledge and accept any potential risks or instability that may arise from modifying these preview features. This process serves as a safeguard, ensuring that you are fully aware of the experimental nature of the flags you are working with.
```

File: docs/content/preview/reference/configuration/yb-master.md [hunk 2]
```diff
@@ -983,6 +983,16 @@ expensive when the number of yb-tservers, or the number of databases goes up.
 
 ## Advanced flags
 
+##### --allowed_preview_flags_csv
+
+This is a comma-separated values (CSV) formatted catalogue of [preview feature](../../../releases/versioning/#tech-preview-tp) flag names. Preview flags represent experimental or in-development features that are not yet fully supported. Flags that are tagged as Preview cannot be modified or configured unless they are included in this list.
```
Comment (aishwarya24) [inline docs/content/preview/reference/configuration/yb-master.md:988]:
```suggestion
This is a comma-separated values (CSV) formatted catalogue of [preview feature](../../../releases/versioning/#tech-preview-tp) flag names. Preview flags represent experimental or in-development features that are not yet fully supported. Flags that are tagged as "preview" cannot be modified or configured unless they are included in this list.
```

**Review by aishwarya24 — APPROVED** [review]
LGTM. Can you also copy the "--allowed_preview_flags_csv" flag info to stable?

File: docs/content/preview/releases/versioning.md [hunk 3]
```diff
@@ -98,6 +98,10 @@ TP features aren't supported by Customer Support and may change at any time duri
 
 If you are interested in a TP feature, contact {{% support-general %}}.
 
+{{<tip title="Enabling preview features">}}
+Configuration of preview feature-specific flags is restricted to those listed in the [allowed_preview_flags_csv](../../reference/configuration/yb-master/#allowed-preview-flags-csv). This measure acts as a safeguard, ensuring users are consciously aware they are modifying settings for preview features.
```
Comment (ddhodge) [inline docs/content/preview/releases/versioning.md:102]:
```suggestion
Configuration of preview feature-specific flags is restricted to those listed in the [allowed_preview_flags_csv](../../reference/configuration/yb-master/#allowed-preview-flags-csv) flag. This measure acts as a safeguard, ensuring users are aware they are modifying settings for preview features.
```

File: docs/content/preview/yugabyte-platform/alerts-monitoring/universe-logging.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ The Audit log export feature is {{<badge/ea>}}. To enable the feature in Yugabyt
 
 To export logs, you need to first create an export configuration. A configuration defines the sign in credentials and settings for the tool that you want to export your logs to. Refer to [Manage export configurations](../anywhere-export-configuration).
 
-If you want to set pgaudit.log_level to a [severity level](https://www.postgresql.org/docs/11/runtime-config-logging.html#RUNTIME-CONFIG-SEVERITY-LEVELS) greater than WARNING (that is, DEBUG1-5, INFO, NOTICE), you must set the YB-TServer [ysql_log_min_messages](../../reference/configuration/yb-tserver/#--ysql-log-min-messages) flag accordingly.
+If you want to set pgaudit.log_level to a [severity level](https://www.postgresql.org/docs/11/runtime-config-logging.html#RUNTIME-CONFIG-SEVERITY-LEVELS) greater than WARNING (that is, DEBUG1-5, INFO, NOTICE), you must set the YB-TServer [ysql_log_min_messages](../../../reference/configuration/yb-tserver/#--ysql-log-min-messages) flag accordingly.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/alerts-monitoring/universe-logging.md:28]:
```suggestion
If you want to set pgaudit.log_level to a [severity level](https://www.postgresql.org/docs/11/runtime-config-logging.html#RUNTIME-CONFIG-SEVERITY-LEVELS) greater than WARNING (that is, DEBUG1-5, INFO, NOTICE), you must set the YB-TServer [ysql_log_min_messages](../../../reference/configuration/yb-tserver/#ysql-log-min-messages) flag accordingly.
```

**Pull request merged**